specification/compute/resource-manager/Microsoft.Compute/stable/2020-05-01/disk.json

```diff
@@ -1475,8 +1843,7 @@
         }
       },
       "required": [
-        "keyUrl",
-        "sourceVault"
```

Comment:
@Chetan. can you please remove this change. This will have to go in the next api version

Reply:
ok